lib/generators/active_fedora/config/solr/templates/solr/config/solrconfig.xml

```diff
@@ -102,6 +102,7 @@
        -->
         <str name="qf">
           id
+          active_fedora_model_ssi
```

Comment:
I wonder if this is cruft. `active_fedora_model_ssi` does not seem to be widely used; I could only find two references to that field in Sufia's dependencies:

 * https://github.com/projecthydra/curation_concerns/blob/master/spec/views/curation_concerns/base/_member.html.erb_spec.rb#L6
 * https://github.com/projecthydra/hydra-head/blob/master/hydra-core/spec/helpers/blacklight_helper_spec.rb#L23-L25